forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -507,3 +507,36 @@ def export(  # pylint:disable=too-many-locals
             }
 
         raise BusinessException(BusinessErrorCode.INVALID_FORM_PROCESS_MAPPER_ID)
+
+    @staticmethod
+    def validate_form_name_path_title(request):
+        """Validate a form name by calling the external validation API."""
+        # Retrieve the parameters from the query string
+        title = request.args.get("title")
+        name = request.args.get("name")
+        path = request.args.get("path")
+        form_id = request.args.get("id")
+
+        # Check if at least one query parameter is provided
+        if not (title or name or path):
+            return {"message": "At least one query parameter (title, name, path) must be provided."}
```

Comment:
Can we use the same format we use for validation errors ? With message and key in same schema ?